packages/size-limit/index.d.ts

```diff
@@ -0,0 +1,92 @@
+export type Config = {
```

Comment:
```suggestion
export interface Config {
```

`interface` is a little faster than `type`, as I know

Reply:
Though shouldn't I rename `Config` to `Options` instead of `Check`?